Contributors Leaderboard added in Navbar

utkarshrastogi121 opened the pull request:
---
name: "📦 Pull Request"
about: Submit changes for review
title: "PR: Added Contributors Leaderboard"
labels: ""
assignees: ""
---

## 📌 Linked Issue
- [x] Connected to #104
---

Roshansuthar1105 commented:
@utkarshrastogi121 add leaderboard link in both navbar mobile and laptop version.
- Also make small devices ui like codify's leaderboard https://codifylearn.netlify.app/

Roshansuthar1105 commented:
@utkarshrastogi121 from next time make a detailed pull request, which is understandable.

utkarshrastogi121 commented:
@Roshansuthar1105 i have updated the responsive UI for mobile screen
Also i have added the leaderboard link for mobile screen as well.

You can check and suggest if some changes are required.
